### ISSUE TITLE
Fix CI

### ISSUE THREAD

**Issue by placccebo** [bwsw/t-streams#754]
find out the reason(s) for CI failures. if several of them are not easy to fix at this stage then create a task for each (commenting a test for a while)

**Comment by tomskikh** [comment]
Failing tests:
```
CommonCheckpointGroupServerTtlTest:
Expired ledgers
- should be deleted according to settings if a server works in a stable way *** FAILED *** 
  org.apache.zookeeper.KeeperException$NoNodeException: KeeperErrorCode = NoNode for /tree/common/5c1db723-57bb-4e49-87d1-978174135755/0/0/0/4
  at org.apache.zookeeper.KeeperException.create(KeeperException.java:114)
  at org.apache.zookeeper.KeeperException.create(KeeperException.java:54)
  at org.apache.zookeeper.ZooKeeper.getData(ZooKeeper.java:1215)
  at org.apache.curator.framework.imps.GetDataBuilderImpl$4.call(GetDataBuilderImpl.java:310)
  at org.apache.curator.framework.imps.GetDataBuilderImpl$4.call(GetDataBuilderImpl.java:299)
  at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:109)
  at org.apache.curator.framework.imps.GetDataBuilderImpl.pathInForeground(GetDataBuilderImpl.java:296)
  at org.apache.curator.framework.imps.GetDataBuilderImpl.forPath(GetDataBuilderImpl.java:287)
  at org.apache.curator.framework.imps.GetDataBuilderImpl.forPath(GetDataBuilderImpl.java:34)
  at com.****.tstreamstransactionserver.netty.server.multiNode.bookkeeperService.hierarchy.ZookeeperTreeList.getNextNode(ZookeeperTreeList.scala:103)
```
```
CommonCheckpointGroupServerTtlTest:
Expired ledgers
- should be deleted according to settings if a server works in a stable way *** FAILED ***
  true was not equal to false (CommonCheckpointGroupServerTtlTest.scala:108)
```
```
SubscriberTransactionPartitionDistributionTest:
Subscriber
Exception in thread "pe-1639816076-executor" com.****.tstreamstransactionserver.exception.Throwable$ZkNoConnectionException: Can't connect to ZooKeeper server(s): localhost:21810!
....at com.****.tstreamstransactionserver.netty.client.zk.ZKMasterPathMonitor.eventReceived(ZKMasterPathMonitor.scala:124)
....at org.apache.curator.framework.imps.CuratorFrameworkImpl$8.apply(CuratorFrameworkImpl.java:888)
....at org.apache.curator.framework.imps.CuratorFrameworkImpl$8.apply(CuratorFrameworkImpl.java:881)
....at org.apache.curator.framework.listen.ListenerContainer$1.run(ListenerContainer.java:93)
....at org.apache.curator.shaded.com.google.common.util.concurrent.MoreExecutors$SameThreadExecutorService.execute(MoreExecutors.java:297)
....at org.apache.curator.framework.listen.ListenerContainer.forEach(ListenerContainer.java:85)
....at org.apache.curator.framework.imps.CuratorFrameworkImpl.processEvent(CuratorFrameworkImpl.java:880)
....at org.apache.curator.framework.imps.CuratorFrameworkImpl.access$000(CuratorFrameworkImpl.java:64)
....at org.apache.curator.framework.imps.CuratorFrameworkImpl$1.process(CuratorFrameworkImpl.java:109)
....at org.apache.curator.ConnectionState.process(ConnectionState.java:178)
....at org.apache.zookeeper.ClientCnxn$EventThread.processEvent(ClientCnxn.java:531)
....at org.apache.zookeeper.ClientCnxn$EventThread.run(ClientCnxn.java:506)
- should retrieve transaction from correct partitions *** FAILED ***
  false was not equal to true (SubscriberTransactionPartitionDistributionTest.scala:75)
```

**Comment by tomskikh** [comment]
Curator client lost connection with ZooKeeper server, ZKMasterPathMonitor throws exception.

```
198584 [pool-1-thread-1-ScalaTest-running-ProducersComeAndLeaveConsumerReadsTest-SendThread(localhost.z1.netpoint-dc.com:21810)] WARN  org.apache.zookeeper.ClientCnxn  - Client session timed out, have not heard from server in 3333ms for sessionid 0x1005090d0440052
198584 [pool-1-thread-1-ScalaTest-running-ProducersComeAndLeaveConsumerReadsTest-SendThread(localhost.z1.netpoint-dc.com:21810)] INFO  org.apache.zookeeper.ClientCnxn  - Client session timed out, have not heard from server in 3333ms for sessionid 0x1005090d0440052, closing socket connection and attempting reconnect
198584 [NIOServerCxn.Factory:0.0.0.0/0.0.0.0:21810] WARN  org.apache.zookeeper.server.NIOServerCnxn  - Unable to read additional data from client sessionid 0x1005090d0440052, likely client has closed socket
198584 [NIOServerCxn.Factory:0.0.0.0/0.0.0.0:21810] INFO  org.apache.zookeeper.server.NIOServerCnxn  - Closed socket connection for client /127.0.0.1:43928 which had sessionid 0x1005090d0440052
198684 [pool-1-thread-1-ScalaTest-running-ProducersComeAndLeaveConsumerReadsTest-EventThread] INFO  org.apache.curator.framework.state.ConnectionStateManager  - State change: SUSPENDED
198685 [pool-1-thread-1-ScalaTest-running-ProducersComeAndLeaveConsumerReadsTest-EventThread] INFO  com.bwsw.tstreamstransactionserver.netty.client.NettyConnection  - Close a connection to 127.0.0.1:8071
198686 [pool-1-thread-1-ScalaTest-running-ProducersComeAndLeaveConsumerReadsTest-EventThread] ERROR org.apache.curator.framework.imps.CuratorFrameworkImpl  - Event listener threw exception
com.bwsw.tstreamstransactionserver.exception.Throwable$ZkNoConnectionException: Can't connect to ZooKeeper server(s): localhost:21810!
	at com.bwsw.tstreamstransactionserver.netty.client.zk.ZKMasterPathMonitor.eventReceived(ZKMasterPathMonitor.scala:124)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl$8.apply(CuratorFrameworkImpl.java:888)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl$8.apply(CuratorFrameworkImpl.java:881)
	at org.apache.curator.framework.listen.ListenerContainer$1.run(ListenerContainer.java:93)
	at org.apache.curator.shaded.com.google.common.util.concurrent.MoreExecutors$SameThreadExecutorService.execute(MoreExecutors.java:297)
	at org.apache.curator.framework.listen.ListenerContainer.forEach(ListenerContainer.java:85)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.processEvent(CuratorFrameworkImpl.java:880)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.access$000(CuratorFrameworkImpl.java:64)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl$1.process(CuratorFrameworkImpl.java:109)
	at org.apache.curator.ConnectionState.process(ConnectionState.java:178)
	at org.apache.zookeeper.ClientCnxn$EventThread.processEvent(ClientCnxn.java:531)
	at org.apache.zookeeper.ClientCnxn$EventThread.run(ClientCnxn.java:506)
198687 [pool-1-thread-1-ScalaTest-running-ProducersComeAndLeaveConsumerReadsTest-EventThread] INFO  com.bwsw.tstreamstransactionserver.netty.client.NettyConnection  - Close a connection to 127.0.0.1:8071
198687 [pool-1-thread-1-ScalaTest-running-ProducersComeAndLeaveConsumerReadsTest-EventThread] ERROR org.apache.curator.framework.imps.CuratorFrameworkImpl  - Event listener threw exception
com.bwsw.tstreamstransactionserver.exception.Throwable$ZkNoConnectionException: Can't connect to ZooKeeper server(s): localhost:21810!
	at com.bwsw.tstreamstransactionserver.netty.client.zk.ZKMasterPathMonitor.eventReceived(ZKMasterPathMonitor.scala:124)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl$8.apply(CuratorFrameworkImpl.java:888)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl$8.apply(CuratorFrameworkImpl.java:881)
	at org.apache.curator.framework.listen.ListenerContainer$1.run(ListenerContainer.java:93)
	at org.apache.curator.shaded.com.google.common.util.concurrent.MoreExecutors$SameThreadExecutorService.execute(MoreExecutors.java:297)
	at org.apache.curator.framework.listen.ListenerContainer.forEach(ListenerContainer.java:85)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.processEvent(CuratorFrameworkImpl.java:880)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.access$000(CuratorFrameworkImpl.java:64)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl$1.process(CuratorFrameworkImpl.java:109)
	at org.apache.curator.ConnectionState.process(ConnectionState.java:178)
	at org.apache.zookeeper.ClientCnxn$EventThread.processEvent(ClientCnxn.java:531)
	at org.apache.zookeeper.ClientCnxn$EventThread.run(ClientCnxn.java:506)
198732 [SyncThread:0] WARN  org.apache.zookeeper.server.persistence.FileTxnLog  - fsync-ing the write ahead log in SyncThread:0 took 1945ms which will adversely effect operation latency. See the ZooKeeper troubleshooting guide
```

**Comment by tomskikh** [comment]
https://github.com/bwsw/t-streams/blob/be2903996ae9a2dd08e0cb0351d7ab07482e266d/tstreams-transaction-server/src/main/scala/com/bwsw/tstreamstransactionserver/netty/client/InetClientProxy.scala#L109
